config/config.yml.example

```diff
@@ -68,20 +68,20 @@ development: &default
   # Generate a key with SecureRandom to enable two-factor authentication (2FA) `SecureRandom.hex(20)`
   # If blank the QRCode won't be generated and the 2FA can't be enabled
   #
-  two_factor_key: # '<Google two factor key>'
+  two_factor_key: "a2ab0425f2cfa4243110e343bc865653aea941b91819e754c9a82a0115266add" # '<Google two factor key>'
```

Comment:
This is just a random string not used anywhere else, right? :)

Reply:
Yes, it was generated using `SecureRandom`